saleor/graphql/account/mutations/authentication/request_password_reset.py

```diff
@@ -72,58 +69,31 @@ def clean_user(cls, email, redirect_url, info: ResolveInfo):
             )
 
         user = retrieve_user_by_email(email)
-        if not user:
-            raise ValidationError(
-                {
-                    "email": ValidationError(
-                        "User with this email doesn't exist",
-                        code=AccountErrorCode.NOT_FOUND.value,
-                    )
-                }
-            )
-
-        if not user.is_active:
-            raise ValidationError(
-                {
-                    "email": ValidationError(
-                        "User with this email is inactive",
-                        code=AccountErrorCode.INACTIVE.value,
-                    )
-                }
-            )
-
-        if password_reset_time := user.last_password_reset_request:
-            delta = timezone.now() - password_reset_time
+        if user and user.last_password_reset_request:
+            delta = timezone.now() - user.last_password_reset_request
             if delta.total_seconds() < settings.RESET_PASSWORD_LOCK_TIME:
-                raise ValidationError(
-                    {
-                        "email": ValidationError(
-                            "Password reset already requested",
-                            code=AccountErrorCode.PASSWORD_RESET_ALREADY_REQUESTED.value,
-                        )
-                    }
-                )
+                user = None
 
         return user
 
     @classmethod
     def perform_mutation(cls, _root, info: ResolveInfo, /, **data):
         email = data["email"]
         redirect_url = data["redirect_url"]
-        user = cls.clean_user(email, redirect_url, info)
+        user = cls.clean_user(email, redirect_url)
         channel_slug = data.get("channel")
-        token = default_token_generator.make_token(user)
-        params = urlencode({"email": user.email, "token": token})
 
-        if not user.is_staff:
```

Comment:
This check is removed otherwise we would allow to enumerate through staff users (only)